src/OSSupport/HostnameLookup.cpp

```diff
@@ -5,62 +5,78 @@
 
 #include "Globals.h"
 #include "HostnameLookup.h"
-#include <event2/dns.h>
 #include "NetworkSingleton.h"
 
 
 
 
 
+static const char * ErrorString(int a_ErrorCode)
+{
+	// Avoid the windows API wide character version
+#ifdef _WIN32
+	return gai_strerrorA(a_ErrorCode);
```

Comment:
From MSDN:
 > Note that the gai_strerror function is not thread safe, and therefore, use of traditional Windows Sockets functions such as the WSAGetLastError function is recommended.

https://msdn.microsoft.com/en-us/library/windows/desktop/ms738514(v=vs.85).aspx

Since there's already the `#ifdef _WIN32`, we could as well use the WSA function.

Also note that we indent even preprocessor code.